include/SFML/Graphics/Color.inl

```diff
@@ -42,10 +42,10 @@ a(alpha)
 
 ////////////////////////////////////////////////////////////
 constexpr Color::Color(std::uint32_t color) :
-r(static_cast<std::uint8_t>((color & 0xff000000) >> 24)),
-g(static_cast<std::uint8_t>((color & 0x00ff0000) >> 16)),
-b(static_cast<std::uint8_t>((color & 0x0000ff00) >> 8)),
-a(static_cast<std::uint8_t>(color & 0x000000ff))
+r(static_cast<std::uint8_t>((color & 0xff'00'00'00) >> 24)),
+g(static_cast<std::uint8_t>((color & 0x00'ff'00'00) >> 16)),
+b(static_cast<std::uint8_t>((color & 0x00'00'ff'00) >> 8)),
+a(static_cast<std::uint8_t>((color & 0x00'00'00'ff) >> 0))
```

Comment:
I definitely approve of all the changes to integers but I'm not yet sure what I think about applying to hexidecal literals. I'm not sure if I don't like or if it's just unfamiliar. I suppose all the same arguments apply to hex values.

Reply:
In general I don't think *every* hexadecimal constant benefits from digit separators.

I picked the hexadecimal constants where doublets have a meaning (e.g. in the case above the each doublet is the value of a channel). I think it helps to visually separate those, as a potential typo/mistake such as `0x00000ff` is hard to spot compared to `0x00'00'0f'f`.